scripts/common.sh

```diff
@@ -21,11 +21,13 @@ resolveCommand () {
   fi
 }
 
+set +u
 if [[ -z $CONTAINER_ENGINE ]]; then
   echo "Setting CONTAINER_ENGINE to default: docker"
   CONTAINER_ENGINE=$(resolveCommand docker)
   export CONTAINER_ENGINE
 fi
+set -u
```

Comment:
```suggestion
CONTAINER_ENGINE=$(resolveCommand ${CONTAINER_ENGINE:docker)
export CONTAINER_ENGINE
```

That way we don't actually need to care?

Reply:
it complains about unbound with that defaulting syntax too:

```
[roby@roby kroxylicious]$ QUAY_ORG=robeyoun ./scripts/run-with-strimzi.sh kubernetes-examples/portperbroker_plain
/home/roby/development/redhat-managed-kafka/upstream/kroxylicious/scripts/common.sh: line 24: CONTAINER_ENGINE: unbound variable
```

Reply:
sorry maybe I got it wrong, I think it works once we get the dash in there `${CONTAINER_ENGINE:-docker}`

Reply:
Oh whoops my bad.

Reply:
No thanks for pointing at that, I think it's better in both cases than toggling the shell options. I changed the other usage to `${QUAY_ORG:-}` to default it to empty string.